pkg/azure/api/api.go

```diff
@@ -19,6 +20,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v7"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v7"
+	"github.com/sirupsen/logrus"
```

Comment:
i think all cilium packages are now using slog. can we use the same?

Reply:
sure. I can change this.

Reply:
this is changed